autosklearn/experimental/askl2.py

```diff
@@ -371,13 +371,16 @@ def fit(self, X, y,
                 'mlp',
             ]
         self.include['classifier'] = include_estimators
+        self.automl_._include['classifier'] = include_estimators
```

Comment:
This is due to not building `automl_` on fit, we have to set this manually as we do for the estimator.